service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReviewCMOControllerAboutToSubmitTest.java

```diff
@@ -160,6 +157,30 @@ void shouldNotUpdateStateToIssueResolutionWhenReviewDecisionIsNotSendToAllPartie
         assertThat(State.ISSUE_RESOLUTION).isNotEqualTo(responseData.getState());
     }
 
+    @Test
+    void shouldNotUpdateStateToIssueResolutionWhenNextHearingIsNotOfTypeIssueResolution() throws Exception {
+        given(documentConversionService.convertToPdf(cmo.getOrder())).willReturn(convertedDocument);
+        given(documentSealingService.sealDocument(convertedDocument)).willReturn(sealedDocument);
```

Comment:
these could also go in the setup  and removed from the three methods
 given(documentConversionService.convertToPdf(cmo.getOrder())).willReturn(convertedDocument);
 given(documentSealingService.sealDocument(convertedDocument)).willReturn(sealedDocument);